### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_Jobs_Keyboard:Table drop down of header and "sort ascending"and "sort descending" button is not accessable by keybaord

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5668]
**Bug Description:**
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
[11:40 AM] Chitrarupa Sharma (Tata Consultancy Services Ltd)
    
GitHubTags: #A11yMAS; #A11yTCS; #MAS2.1.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**

Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
App Version: 1.8.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
**Additional Details:**
Tool Name: tab 
MAS Reference:MAS2.1.1

**Repro Steps:**
1. Install the app Azure Data Studio.
2. Go to the  SQL server agent tab->Select Jobs tab.
3. Navigate till ""New job"" button
4.Observe the navigation till ""New job"" button 
5.After New job button navigate till table header dropdown arrow by mouse as it is not accessible by key board.
6.click on dropdown to open the popup.
7.go on ""new job"" button hit tab .
**Actual:**
Table header drop down  and ""sort ascending" "and ""sort descending"" button is not accessible by keyboard.
focus  directly jump on search edit box rather then going on ""sort ascending" "and ""sort descending"" button.
This is also observed with Win10.

**Expected:**
keyboard Focus should go on ""sort ascending" "and ""sort descending"" button after drop down header.

**User Impact:**
User with disabilities find it difficult to navigate the page.
Recommendation:
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference**:
**MAS2.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

"
**Attachment for reference:-**
[5668_AccessibilityA11y_AzureDataStudio_Jobs_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3225967/5668_AccessibilityA11y_AzureDataStudio_Jobs_Keyboard.pptx)

**Comment by AccessibilityTestingTeam-TCS** [comment]
**Verified in the below environment:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8:6.6.346.32
OS : Windows_NT x64 10.0.18362
Tool Name: NVDA/jaws
Bug has not been fixed it is still repro hence reopening the bug.
**Attachment for reference:-**
[5668_Reopened_A11y_AzureDataStudio_Alljobs_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3285581/5668_Reopened_A11y_AzureDataStudio_Alljobs_Screenreader.pptx)
